src/components/select/filterable-select/filterable-select.spec.js

```diff
@@ -433,6 +434,16 @@ describe('when the component is controlled', () => {
       expect(onChangeFn).toHaveBeenCalledWith(expectedObject);
     });
   });
+
+  it('the required prop is passed to the input', () => {
+    const wrapper = renderSelect({ label: 'required', required: true });
+
+    const input = wrapper.find('input');
+    expect(input.prop('required')).toBe(true);
+
+    const label = wrapper.find(Label);
+    expect(label.prop('isRequired')).toBe(true);
+  });
```

Comment:
two tests in one case

Reply:
Updated test description because the setup is the same.